CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Display of error payloads in console event log.
 - Zero coordinate handling in location form in the Console.
+- Support for multiple resulting messages from one format
```

Comment:
This should be added to the Unreleased part, as it is not released yet, and not part of 3.5.3. Also add a trailing dot.